chainquery/Dockerfile

```diff
@@ -1,52 +1,24 @@
-## This base image is for running latest chainquery
+## Get the latest source and extract it for the app container.
+## Design choices, two RUN layers intended to keep builds faster, the zipped
 FROM ubuntu:18.04 as prep
 LABEL MAINTAINER="leopere [at] nixc [dot] us"
-
-## Install everything needed to unzip the file containing Chainquery
 RUN apt-get update && \
-    apt-get -y install unzip curl && \
-    apt-get autoclean -y && \
-    rm -rf /var/lib/apt/lists/* && \
-    mkdir /download
-
-## Download and extract the latest Zip containing Linux binary for Chainquery.
-RUN curl -L -o /download/chainquery.zip $(curl -s https://api.github.com/repos/lbryio/chainquery/releases | fgrep 'Linux_x86_64.zip' | grep download | head -n 1 | cut -d'"' -f4) && \
-    cd /download && \
-    ls -lAh /download && pwd && \
-    unzip chainquery.zip && \
-    rm chainquery.zip
-
-## TODO: Use this instead of everything above here at some point. To do this we will need the LBRY team to host all of their release binaries in their latest form internally I guess since github doesn't support latest tags.
-# ADD --chown=1000:1000 https://github.com/lbryio/chainquery/releases/download/latest/chainquery_latest_Linux_x86_64.zip /download
-
+  apt-get -y install unzip curl && \
+  apt-get autoclean -y && \
+  rm -rf /var/lib/apt/lists/*
+WORKDIR /
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+COPY stuff/start.sh start
+RUN curl -L -o /chainquery.zip $(curl -s https://api.github.com/repos/lbryio/chainquery/releases \
```

Comment:
This line is throwing a linting error about word splitting but I think it's fine.  I don't really want to fix this at the moment and so long as it runs its an acceptable error.